modules/update-preparing-evaluate-alerts.adoc

```diff
@@ -10,4 +10,6 @@ Two alerts fire when an API is in use that will be removed in the next release:
 * `APIRemovedInNextReleaseInUse` - for APIs that will be removed in the next {product-title} release.
 * `APIRemovedInNextEUSReleaseInUse` - for APIs that will be removed in the next {product-title} Extended Update Support (EUS) release.
 
-If either of these alerts are firing in your cluster, review the alerts and take action to clear the alerts by migrating manifests and API clients to use the new API version. You can use the `APIRequestCount` API to get more information about which APIs are in use and which workloads are using removed APIs.
+If either of these alerts are firing in your cluster, review the alerts and take action to clear the alerts by migrating manifests and API clients to use the new API version.
+
+Use the `APIRequestCount` API to get more information about which APIs are in use and which workloads are using removed APIs, because the alerts do not provide this information. Additionally, some APIs might not trigger these alerts but are still be captured by `APIRequestCount`. The alerts are tuned to not be overly sensitive to avoid alerting fatigue in production systems.
```

Comment:
```suggestion
Use the `APIRequestCount` API to get more information about which APIs are in use and which workloads are using removed APIs, because the alerts do not provide this information. Additionally, some APIs might not trigger these alerts but are still being captured by `APIRequestCount`. The alerts are tuned to not be overly sensitive to avoid alerting fatigue in production systems.
```

Reply:
Consider s/to not be overly sensitive/to be less sensitive

Reply:
Good suggestions, thanks!